docs/release-notes/2.30.0.md

```diff
@@ -0,0 +1,138 @@
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+#### New Experimental dashboards
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards aresubject to change. It is recommended to use these dashboards for testing purposes only.
+
+- We have introduced a more user-friendly and insightful experimental Home Dashboard in PMM version 2.30.0. With this new Home dashboard, you can experience a boost in performance while monitoring a large number of nodes or services in PMM. Repeated rows (for a large number of nodes) on the dashboard can take a considerable time to load the Home Dashboard UI. Our new experimental dashboard ensures fast retrieval of data on the Home dashboard.
+
+- We have introduced the following new real-time experimental dashboards to collect more metrics and data for MongoDB:
+
+    - **Experimental MongoDB Collection Overview**
+        
+        This dashboard contains panels for data about the Hottest Collections in the MongoDB database.
+
+    - **Experimental MongoDB Collection Details**
+
+        This experimental dashboard provides detailed information about the top collections by document count, size, and document read for MongoDB databases.
+
+    - **Experimental MongoDB Oplog Details**
+
+        This real-time dashboard contains Oplog details such as Recovery Window, Processing Time, Buffer Capacity, and Oplog Operations.
+
+
+### Database as a Service (DBaaS)
+
+Enhanced the user experience by simplifying the configuration and deployment of DBaaS database clusters:
+
+- With PMM 2.30.0, you can create a DB cluster literally at the click of a button. All the fields will be automatically populated with the default value. Click Register, and your DB cluster is created.
+
+- You can easily create a Kubernetes cluster with PMM 2.30.0. When you copy the value of the Kubeconfig file, the Kubernetes Cluster Name gets auto-populated from the contents of the kubeconfig file.
+
+- Starting with PMM 2.23.0, we have simplified the DBaaS configuration and deployment of database clusters by providing suggestions on the k8s cluster name based on the Kubeconfig file.
+
+### Operators support
+
+- PMM now supports Percona Operators MySQL (1.11) and MongoDB (1.12).
+
+- PMM now provides MongoDB v5 deployment support. 
+
+
+
+
+## New Features
+
+- [PMM-9652](https://jira.percona.com/browse/PMM-9652): Dashboard: New experimental dashboards for MongoDB are now available with more MongoDB metrics and data like Hottest collections, Data Size, recovery Window, Processing Time, Buffer Capacity, Oplog Operations, and so on.
```

Comment:
case inconsistency: Hottest collections / Data Size / recovery Window ...

Reply:
Fixed it.